share/man/man4/iwlwifi.4

```diff
@@ -66,190 +68,124 @@ driver which supports older chipsets.
 .Pp
 The driver uses the
 .\" No LinuxKPI man pages so no .Xr here.
-.Em linuxkpi_wlan
+.Sy linuxkpi_wlan
 and
-.Em linuxkpi
+.Sy linuxkpi
 compat framework to bridge between the Linux and
 native
 .Fx
 driver code as well as to the native
 .Xr net80211 4
 wireless stack.
-.Pp
-While
-.Nm
-supports all 802.11 a/b/g/n/ac/ax
-the compatibility code currently only supports 802.11 a/b/g modes.
-Support for 802.11 n/ac is to come. 802.11ax and 6Ghz support are planned.
 .Sh HARDWARE
 The
 .Nm
 driver supports PCIe devices from the
-.Em mvm
+.Sy mvm
 sub-driver with the following chipset generations:
 .Pp
 .\" awk -F\\t '{ print $5 }' ~/tmp/iwlwifi_pci_ids_name.txt | \
 .\" grep -v undefined | sort -V | uniq | grep -v ^$ | \
-.\" awk '{ printf ".It\n%s\n", $0 }'
-.Bl -bullet -compact
-.It
+.\" awk '{ printf "%s\n", $0 }'
+.Bd -literal -offset indent -compact
 7000
-.It
 8000
-.It
 9000
-.It
 22000
-.It
 AX210
-.It
 BZ
-.It
 SC
-.El
+.Ed
 .Pp
 These chipset generations match the following common device names:
-.Pp
-.Bl -bullet -compact
-.\" ---------------------------------------------------------------------
+.Bd -literal -offset indent
+.\" --------------------------------------------------------------------
 .\" This list is manually generated from a sysctl and post-processing.
 .\" Edits will be overwritten on next update.
 .\" awk -F\\t '{ if ($2 == "") { next; } if (seen[$2]) { next; } \
-.\" seen[$2]=1; printf ".It\n%s\n", $2; }' iwlwifi_pci_ids_name.txt
-.\" ---------------------------------------------------------------------
-.It
```

Comment:
Those .It were common in other places; that's where I got them from.
Some of that awk is also in different scripts so please make sure once this goes in I'll review them to make sure they generate the output we want.